### PR TITLE
Feature/veriwell

### PR DESCRIPTION
**Pull request by markjung96**

### CODE REVIEW

File: docs/veriwell/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+title: Getting Started
+description: About multi-language verification tool
+keywords: [Multi chain contract, smart contracts]
+---
+
+### Developer
+
+Developers need to know the contract's address or deployment hash. Additionally, they must be aware of the OS and compiler version used during deployment. Depending on the programming language of the contract, the compilation process may or may not depend on the OS. For example, Arbitrum's Stylus generates different bytecode depending on the OS used during compilation. Similarly, compiler versions can affect compatibility; certain functions may be backward-compatible with earlier versions, while others may not.
+
+As a result, even if the OS and compiler version differ between deployment and verification, successful verification might still be possible in some cases. However, in most scenarios, the verification process fails without a matching environment. Therefore, developers need to know the OS and compiler version used at the time of deployment to replicate the same build and compilation environment, ensuring successful verification.
```
Comment (inchori) [inline docs/veriwell/getting-started.md:12]: I think that this part is adjusted for Arbitrum Stylus. WDYT?

File: docs/veriwell/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Once the contract address or hash and the deployment environment are configured, developers can upload the code for compilation. The code is compiled according to the specified environment, and if the compiled output matches the on-chain code, the contract is verified successfully.
+
+### User
+
+No preparation is required. Simply access the VeriWell website and input the address of the contract you wish to verify. If the contract is verified, you will be redirected to a results page where you can review the contract's code along with explanations provided by AI support about the code's functionality.
```
Comment (inchori) [inline docs/veriwell/getting-started.md:18]: `input the the address of the contract and the source code zip file`?

**Review by inchori — COMMENTED** [review]
Nice!
I left comments

File: docs/veriwell/getting-started.md (hunk 2 above)
Comment (markjung96) [reply docs/veriwell/getting-started.md]: finish editing sentence

File: docs/veriwell/tutorials/staknet/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        fn get(self: @ContractState) -> u128 {
+            self.stored_data.read()
+        }
+    }
+}
+```
+
+### Cairo Verification
+
+The above `simple_storage.cairo` can be verified using WELLDONE Studio's upcoming Multi-chain Verification Tool. To deploy a smart contract written in Cairo, you can refer to the Starknet [documentation](https://docs.starknet.io/quick-start/environment-setup/) for deployment. Once deployed, you can check that it was deployed correctly with [Starkscan](https://starkscan.co/) or [Nethermind Voyager Explorer](https://voyager.online/).
```
[the remark this answers is not in the document]
Comment (markjung96) [reply docs/veriwell/tutorials/staknet/index.md]: sentence editted

**Review by inchori — APPROVED** [review]
LGTM

**Pull request merged**